### ISSUE TITLE
Cant install the package due to node vrtsion restriction

### ISSUE THREAD

**Issue by DmitryEfimenko** [angular-schule/angular-cli-ghpages#162]
I have a restriction on the node defined in the package.json:
```json
"engines": {
  "node": ">=18.10.0"
},
```
When trying to install this package:
> npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for rollup@3.7.5: wanted: {"node":">=14.18.0","npm":">=8.0.0"} (current: {"node":"18.10.0","npm":"6.14.4"})
npm ERR! notsup Not compatible with your version of node/npm: rollup@3.7.5
npm ERR! notsup Not compatible with your version of node/npm: rollup@3.7.5
npm ERR! notsup Required: {"node":">=14.18.0","npm":">=8.0.0"}
npm ERR! notsup Actual:   {"npm":"6.14.4","node":"18.10.0"}

Any help is appreciated

**Comment by JohannesHoppe** [comment]
Good point, it's time to update the dependencies of angular-cli-ghpages sooner than later.

Meanwhile you could try to install another version of rollup manually in your repo, after installing angular-cli-ghpages.
